testsuites/syncgateway/functional/tests/test_db_online_offline_resync.py

```diff
@@ -123,19 +124,27 @@ def test_bucket_online_offline_resync_sanity(params_from_base_test_setup, sg_con
     restart_status = cluster.sync_gateways[0].restart(sg_restart_config,
                                                       cluster_config=cluster_conf)
     assert restart_status == 0
-
+    resync_status = admin.db_resync(db="db")
```

Comment:
we can move these 2 lines under the if, since we are using them only inside the IF statement

Reply:
it is only one line i.e resync_status , put it in else. above line should not be inside if